### DISCUSSION TITLE
Where did the Next.js site source go?

### DISCUSSION THREAD

**Discussion by simonsinclair** [vercel/next.js#18273] [Help] [answered]
Hello Next.js team,

Where did `vercel/next-site` go? This repository was a valuable resource. It's disappearance also makes [links in the official docs](https://nextjs.org/docs/routing/dynamic-routes#optional-catch-all-routes) redundant. 

Kind regards,

Simon

**Comment by Timer** [comment, marked as answer]
Temporarily hidden [for Next.js Conf](https://nextjs.org/conf), it'll return soon!

**Reply by simonsinclair** [reply]
Cool. Thanks, @Timer.

**Reply by MosMK** [reply]
> Temporarily hidden [for Next.js Conf](https://nextjs.org/conf), it'll return soon!

It hasn't returned yet!
